custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/KarafFeatureCheck.java

```diff
@@ -75,25 +75,39 @@ protected void processFiltered(File file, FileText fileText) throws CheckstyleEx
 
             String expression = MessageFormat.format(BINDING_FEATURE_EXPRESSION, bundleId);
 
-            Path featurePath = resolveRecursively(file.toPath(), Paths.get(featureXmlPath));
-            if (featurePath == null) {
-                logger.warn(
-                        this.getClass().getSimpleName() + " will be skipped. Could not find file " + featureXmlPath);
-                return;
-            }
+            String[] individualPaths = featureXmlPath.split(":");
+
+            boolean isFound = false;
+
+            for (int i = 0; i < individualPaths.length; i++) {
+                String singlePath = individualPaths[i];
+
+                Path featurePath = resolveRecursively(file.toPath(), Paths.get(singlePath));
 
-            try {
-                FileText featureFileText = new FileText(featurePath.toFile(), StandardCharsets.UTF_8.name());
-                Document featureXML = parseDomDocumentFromFile(featureFileText);
+                if (featurePath == null) {
+                    logger.warn("Could not find file feature file " + singlePath);
```

Comment:
Does this logger support placeholders? Why don't we use slf4j here? Will this start to be a problem when we enable SAT 0.6.0 on this repo?